mmv1/third_party/terraform/website/docs/guides/common_issues.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+page_title: "Common Issues/FAQ"
```

Comment:
Nit: Let's use sentence case throughout titles/headings here: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Reply:
done